test/unit_tests/molecules/Multiselect-spec.js

```diff
@@ -59,14 +58,18 @@ describe( 'Multiselect', function() {
         var option = document.querySelector( 'option' );
         option.defaultSelected = true;
         multiselect.init();
-        var choices = document.querySelectorAll( '.cf-multi-select_choices li' );
+        var choices =
+          document.querySelectorAll( '.cf-multi-select_choices li' );
 
         expect( choices.length ).to.equal( 1 );
         expect( choices[0].innerHTML ).to.contain( 'Debt collection' );
       }
     );
 
     it( 'should log a helpful tip if passed a bad option value', function() {
+      // Note: sandbox.stub console.log will prevent regular `console.log(…)`
```

Comment:
Could probably just update this to a TODO to match the error one you added in the main file.